common/djangoapps/third_party_auth/tests/specs/test_testshib.py

```diff
@@ -200,6 +201,7 @@ def test_configure_testshib_provider_with_cache_duration(self):
         kwargs.setdefault('enabled', True)
         kwargs.setdefault('visible', True)
         kwargs.setdefault('idp_slug', self.PROVIDER_IDP_SLUG)
```

Comment:
Related to @douglashall's comment, should the old slug fields (`idp_slug`) here be removed?